test/Test/HTTPure/MultiHeadersSpec.purs

```diff
@@ -0,0 +1,133 @@
+module Test.HTTPure.MultiHeadersSpec where
```

Comment:
I get that we're basically duplicating `Test.HTTPure.HeadersSpec` and then updating the relevant bits, but these tests would be easier to read if we followed our convention of fully qualified imports.

In fact, now that we're managing this lib, I feel we should work to bring it under our style guide, though it probably makes more sense to do it all at once in a separate PR, rather than just with the new modules we introduce.